tests/subworkflows/icgc-argo-workflows/merg_sort_dup/test.yml

```diff
@@ -0,0 +1,13 @@
+- name: "merg_sort_dup"
+  command: nextflow run main.nf -entry test_merg_sort_dup --tools 'markdup' -profile docker -c ../../../config/nextflow.config
```

Comment:
Needs a default `--tools` value. 

Running the following :
`nextflow run main.nf -entry test_merg_sort_dup --tools '' -profile docker -c ../../../config/nextflow.config`
results in:
```
N E X T F L O W  ~  version 23.04.1
Launching `main.nf` [mad_banach] DSL2 - revision: 378ebf63df
Missing process or function split([,])

 -- Check script 'main.nf' at line: 47 or see '.nextflow.log' file for more details
```
`nextflow run main.nf -entry test_merg_sort_dup  -profile docker -c ../../../config/nextflow.config`
results in:
```
N E X T F L O W  ~  version 23.04.1
Launching `main.nf` [drunk_meninsky] DSL2 - revision: 378ebf63df
ERROR ~ Cannot invoke method split() on null object

 -- Check script '../../../../subworkflows/icgc-argo-workflows/merg_sort_dup/main.nf' at line: 107 or see '.nextflow.log' file for more details
```
But with:
`nextflow run main.nf -entry test_merg_sort_dup --tools ' ' -profile docker -c ../../../config/nextflow.config`
```
N E X T F L O W  ~  version 23.04.1
Launching `main.nf` [awesome_rubens] DSL2 - revision: 378ebf63df
executor >  local (3)
[78/e030a9] process > test_merg_sort_dup:MERG_SORT_DUP:SAMTOOLS_MERGE (Study123.PatientXYZ.SampleA) [100%] 1 of 1 ✔
[29/590c6c] process > test_merg_sort_dup:MERG_SORT_DUP:SAMTOOLS_INDEX (Study123.PatientXYZ.Sampl... [100%] 1 of 1 ✔
[03/bf5806] process > test_merg_sort_dup:MERG_SORT_DUP:SAMTOOLS_CONVERT (Study123.PatientXYZ.Sam... [100%] 1 of 1 ✔
```